web/modules/custom/dpl_update/dpl_update.install

```diff
@@ -239,3 +240,10 @@ function dpl_update_update_10019(): string {
 
   return $return;
 }
+
+/**
+ * Install typed_data, graphql, graphql_compose and consumers modules.
+ */
+function dpl_update_update_10020(): string {
+  return _dpl_update_install_modules(['typed_data', 'graphql', 'graphql_compose', 'consumers']);
```

Comment:
I have approved the PR. But is it necessary to list all those modules here?
Wouldn't it be enough to eg. include the `graphl_compose` module? - then the rest would follow (didn't double check my assumption).

Reply:
I am actually not sure. I added them all to be on the safe side, as I had problems with this before.